cmd/repo_set_default_test.go

```diff
@@ -22,29 +22,32 @@ import (
 	"github.com/appsody/appsody/cmd/cmdtest"
 )
 
-var repoSetDefaultLogsTests = []struct {
-	testName     string
-	args         []string // input
-	expectedLogs string   // logs that are expected in the output
-}{
-	{"No args", nil, "must specify desired default repository"},
-	{"Existing default repo", []string{"incubator"}, "default repository has already been set to"},
-	{"Non-existing repo", []string{"test"}, "not in your configured list of repositories"},
-	{"Badly formatted repo config", []string{"test", "--config", "testdata/bad_format_repository_config/config.yaml"}, "Failed to parse repository file yaml"},
-}
-
 func TestRepoSetDefaultLogs(t *testing.T) {
+	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
+	defer cleanup()
+
+	var repoSetDefaultLogsTests = []struct {
+		testName     string
+		args         []string // input
+		configDir    string
+		expectedLogs string // logs that are expected in the output
+	}{
+		{"No args", nil, "", "must specify desired default repository"},
+		{"Existing default repo", []string{"incubator"}, "", "default repository has already been set to"},
+		{"Non-existing repo", []string{"test"}, "", "not in your configured list of repositories"},
+		{"Badly formatted repo config", []string{"test"}, "bad_format_repository_config", "Failed to parse repository file yaml"},
+	}
 
 	for _, testData := range repoSetDefaultLogsTests {
 		// need to set testData to a new variable scoped under the for loop
 		// otherwise tests run in parallel may get the wrong testData
 		// because the for loop reassigns it before the func runs
 		tt := testData
 
+		sandbox.SetConfigInTestData(tt.configDir)
+
 		// call t.Run so that we can name and report on individual tests
 		t.Run(tt.testName, func(t *testing.T) {
-			sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
```

Comment:
This should be put back and remove the sandbox up above. The sandbox should be created clean in every `t.Run()`